.github/scripts/unittest-linux/run_test.sh

```diff
@@ -30,6 +30,7 @@ fi
 
 (
     cd test
-    pytest "${args[@]}" torchaudio_unittest
+    export CUBLAS_WORKSPACE_CONFIG=:16:8
```

Comment:
I suspect this may not be needed - at least not for now, since we're not testing on GPU?